### ISSUE TITLE
rename main branch to main; not use master anymore

### ISSUE THREAD

**Issue by jdittrich** [opensourcedesign/opensourcedesign.github.io#307]
As a  'master' branch  connotes the existence of a 'slave' branch (which git itself did never have, _but_, afaic, the system git was modeled on, did), we could change the name of the main branch to… `main`. [This itself is very easy](https://www.hanselman.com/blog/easily-rename-your-git-default-branch-from-master-to-main), but since deployment of our blog or other infrastructure might hinge on the branch name, I rather create this issue first.

**Comment by RDIL** [comment]
True, but does GitHub pages support this yet?  That is a must have, and it would require script/documentation updates.

**Comment by SaptakS** [comment]
I think @jancborchardt is also looking into making the necessary changes related to making the default branch from "master" to "main"

**Comment by jdittrich** [comment]
Do we have a list of what needs to be done, somewhere? It would be helpful anyway to know what all depends on how the main branch is named.

**Comment by RDIL** [comment]
I'm not sure, it might just be changing it in the settings menu.

**Comment by AnXh3L0** [comment]
I have created a new "main" branch derived from the current master branch which is going to be the default one (after we introduce the new redesign of the website).

**Comment by AnXh3L0** [comment]
The new website redesign has been published, and the branch has been changed from master to main, closing as completed!
